tla-types/src/main/scala/at/forsyte/apalache/tla/typecheck/etc/TypeAliasSubstitution.scala

```diff
@@ -1,32 +1,38 @@
 package at.forsyte.apalache.tla.typecheck.etc
 
-import at.forsyte.apalache.tla.lir.{
-  BoolT1, ConstT1, FunT1, IntT1, OperT1, RealT1, RecRowT1, RecT1, RowT1, SeqT1, SetT1, SparseTupT1, StrT1, TlaType1,
-  TupT1, TypingException, UID, VarT1, VariantT1,
-}
+import at.forsyte.apalache.tla.lir._
+import at.forsyte.apalache.tla.typecheck.TypingInputException
 
 /**
  * A substitution from constant names to types. It is very similar to Substitution. However, `ConstSubstitution` is
- * meant to replace constant types, e.g., ENTRY, with a concrete type, whereas `Substitution` replaces variables. We use
- * `ConstSubstitution` to implement aliases.
+ * meant to replace constant types, e.g., ENTRY or $entryInCamlCase, with a concrete type, whereas `Substitution`
+ * replaces variables. We use `ConstSubstitution` to rewrite type aliases.
```

Comment:
Also need to rename the class here :)
(Unfortunately I can't mark the unchanged line above to make a suggestion)

```
 * A substitution from constant names to types. It is very similar to [[Substitution]]. However,
 * [[TypeAliasSubstitution]] is meant to replace constant types, e.g., ENTRY or $entryInCamlCase, with a concrete type,
 * whereas [[Substitution]] replaces variables. We use [[TypeAliasSubstitution]]` to rewrite type aliases.
```